etc/ci_scripts/dump_artifacts.py

```diff
@@ -69,7 +68,7 @@ def dump_all_logs(compose_project: str) -> None:
 
 def dump_volume(compose_project: Optional[str], volume_name: str) -> None:
     # Make a temporary container with the volume mounted
-    # dobi adds a prefix on everything in the docker-compose.
+    # docker-compose prefixes volume names with the compose project name.
```

Comment:
I know we'd discussed just not using volume at all, and `docker cp`'ing from the image or something; where'd we land on that?

Reply:
I decided not to touch this. Thought it might have been necessary, but that's not the case and I didn't want unnecessary scope creep.